recipes/apsw/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "apsw" %}
+{% set sqlite_version = "3.16.2" %}
+{% set version = sqlite_version ~ "-r1" %}
+{% set sha256 = "78d952bfdc8dc6bb6ceb061d506559e4b3992170d00eea2a3b95b4ab40313e41" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version|replace("-", ".") }}
+
+source:
+  fn: {{ version }}.tar.gz
+  url: https://github.com/rogerbinns/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script:
+    - python setup.py fetch --sqlite --version {{ sqlite_version }}  
```

Comment:
No, I didn't. 
Thought it is a good idea, to use a pinned version of sqlite, that is always the same version as apsw. This way apsw is independent of the conda-forge sqlite package.

I could give this a try, if requested ;-)